README.md

```diff
@@ -175,6 +175,9 @@ Once enabled, preloading a checkout is as simple as:
 ShopifyCheckoutSheetKit.preload(checkout: checkoutURL)
 ```
 
+
+On the other hand, calling ShopifyCheckoutSheetKit.preload(checkout: checkoutURL) while preloading=false will cause any calls to the `preload` function to be ignored by the sdk. This is intentional, so you can deactivate preloading on the fly when necessary (eg sudden cpu/memory concerns)
```

Comment:
I would also wrap this part with backtick
`ShopifyCheckoutSheetKit.preload(checkout: checkoutURL)` 

And perhaps 
"while the runtime preloading is set to false" to be more explicit.